bonding.py

```diff
@@ -396,7 +398,7 @@ def peers(quiet=True):
     return groups
 
 
-def automated():
+def automated(peerswait=5):
```

Comment:
There seems to be some inconsistency in use of `wait` vs `peerswait`.  I'd prefer if all of the functions use a more consistent argument name throughout the codebase.  Let's stick with `peerswait`, and not use `wait`.